elasticsearch/utils/es_seed_acl

```diff
@@ -39,3 +43,12 @@ if [ ${now} -ge ${MAX_WAIT_SEED} ]; then
     exit 1
 fi
 info "Seeded the searchguard ACL index"
+
+info "Disabling auto replication"
+sgadmin -dra
+
+info "Updating replica count to 0"
+sgadmin -us 0
+
+info "Setting filters to allocate shard to this node only"
+es_util --query="$index/_settings" -XPUT -d "{\"index.routing.allocation.include._name\": \"$index\"}"
```

Comment:
Should `include` be `require`?

Reply:
I source this information from the ES docs which did not offer an alternative.  Given we anticipate using this for the SG index that is based on the DC name, will this not fundamentally be the same?

Reply:
@jcantrill, do you mean this page?  https://www.elastic.co/guide/en/elasticsearch/reference/2.4/shard-allocation-filtering.html